### PR TITLE
fix statusHeight on non-iphoneX devices in landScape

### PR DESCRIPTION
**Pull request by uragecz**
Problem in landScape on non-iPhoneX devices. There was no statusBar height. In landScape mode it returned 0.

screen shot 2018-01-30 at 11 42 43

### CODE REVIEW

File: src/views/SafeAreaView.js [hunk 1]
```diff
@@ -76,11 +76,7 @@ const statusBarHeight = isLandscape => {
     return isLandscape ? 0 : 44;
   }
 
-  if (isIPad) {
```
Comment (ericvicenti) [inline src/views/SafeAreaView.js]: Why did you delete this? I think iPad landscape mode still has 20px status bar

Comment (uragecz) [reply src/views/SafeAreaView.js]: Because its unnecessary part of code... now we always return 20px..

Comment (uragecz) [reply src/views/SafeAreaView.js]: And the test is unnecessary too... all non-iPhone X devices have status bar, only iPhone X in landscape mode doesn't... so it works

Comment (ericvicenti) [reply src/views/SafeAreaView.js]: Oops, I misread that. So, its always 20px? What about smaller phone sizes or older versions of iOS? If you think this is safe then lets merge!

Comment (uragecz) [reply src/views/SafeAreaView.js]: Ehm, i just found it was some bug on my simulator... because the status bar should hide on landscape mode.. so your code is correct, i apologize
